src/submission/SubmissionFileBlacklist.js

```diff
@@ -34,11 +34,20 @@ class SubmissionFileBlacklist {
       if (fileLine.startsWith("#") || fileLine.length === 0) {
         return [...globs]
       } else {
-        const fileOrDir = fs.statSync(path.join(this.directory, fileLine))
-        if (fileOrDir.isDirectory()) {
-          fileLine = this._formatDirectoryGlob(fileLine)
+        try {
+          const fileOrDir = fs.statSync(path.join(this.directory, fileLine))
+          // console.log("File exists.");
+
+          if (fileOrDir.isDirectory()) {
+            fileLine = this._formatDirectoryGlob(fileLine)
+          }
+          return [...globs, fileLine]
+        }
+        catch (error) {
```

Comment:
we should check specifically for the `ENOENT` error and throw otherwise.

Reply:
the `try` should really only be around the `statSync` call because that is where we anticipate the error to occur